src/zenml/stack/stack.py

```diff
@@ -233,6 +233,15 @@ def runtime_options(self) -> Dict[str, Any]:
 
         return runtime_options
 
+    def dict(self) -> Dict[str, str]:
```

Comment:
should this be to `to_dict` to avoid clashes with the internal keyword `dict`?

Reply:
We have shortly discussed it with @schustmi and decided that it would be nice to stay consistent with the pydantic elements within the stack.

Reply:
👍

Reply:
👍